src/main.cpp

```diff
@@ -179,7 +180,7 @@ void DDWAF::update_required_addresses(const Napi::CallbackInfo& info) {
   Napi::Env env = info.Env();
 
   uint32_t size = 0;
-  const char* const* required_addresses = ddwaf_required_addresses(this->_handle, &size);
+  const char* const* required_addresses = ddwaf_known_addresses(this->_handle, &size);
```

Comment:
why are we maintaining the `required_addresses` variable name, if it is not called required_address anymore?

Reply:
You're right.